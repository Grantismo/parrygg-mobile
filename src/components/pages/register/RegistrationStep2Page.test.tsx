import { describe, expect, it } from "@jest/globals";
import React from "react";
import renderer from "react-test-renderer";

import RegistrationStep2Page from "@/components/pages/register/RegistrationStep2Page";

describe("RegistrationStep2Page", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<RegistrationStep2Page />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
