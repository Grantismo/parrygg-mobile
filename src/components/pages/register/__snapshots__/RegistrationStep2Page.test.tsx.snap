// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationStep2Page renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#1b1b1b",
    }
  }
>
  <View
    style={
      {
        "height": "100%",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <Image
      source={1}
      style={
        {
          "opacity": 0.03,
          "resizeMode": "cover",
        }
      }
    />
  </View>
  <RNCSafeAreaView
    edges={
      {
        "bottom": "additive",
        "left": "additive",
        "right": "additive",
        "top": "additive",
      }
    }
    style={
      [
        {
          "display": "flex",
          "flexDirection": "column",
          "height": "100%",
        },
      ]
    }
  >
    <View>
      <View
        style={
          [
            {
              "height": "100%",
              "paddingLeft": 24,
              "paddingRight": 24,
            },
            [
              {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
                "justifyContent": "center",
              },
              undefined,
            ],
          ]
        }
      >
        <Text
          style={
            [
              {
                "fontFamily": "Inter_600SemiBold",
              },
              {
                "color": "#FFC93F",
              },
              {
                "fontSize": 20,
                "lineHeight": 28,
              },
              {
                "marginBottom": 24,
              },
            ]
          }
        >
          Set up 
          <Text
            style={
              [
                {
                  "fontFamily": "Inter_600SemiBold",
                },
                {
                  "color": "#fff",
                },
                {
                  "fontSize": 20,
                  "lineHeight": 28,
                },
                undefined,
              ]
            }
          >
            Location
          </Text>
        </Text>
        <View
          style={
            [
              {
                "marginBottom": 24,
                "width": "100%",
              },
              false,
            ]
          }
        >
          <View
            onLayout={[Function]}
            style={
              {
                "justifyContent": "center",
              }
            }
          >
            <ViewManagerAdapter_ExpoLinearGradient
              colors={
                [
                  4278979596,
                  4279637526,
                ]
              }
              style={
                {
                  "borderRadius": 12,
                  "position": "relative",
                  "zIndex": 20,
                }
              }
            >
              <Text
                style={
                  [
                    {
                      "fontFamily": "Inter_500Medium",
                    },
                    {
                      "fontSize": 16,
                    },
                    {
                      "color": "#fff",
                    },
                    {
                      "backgroundColor": "#161616",
                      "borderRadius": 10,
                      "color": "#fff",
                      "fontSize": 12,
                      "fontWeight": "normal",
                      "marginLeft": 16,
                      "paddingBottom": 1.6,
                      "paddingLeft": 7.2,
                      "paddingRight": 7.2,
                      "paddingTop": 1.6,
                      "position": "absolute",
                      "top": -12,
                      "zIndex": 20,
                    },
                  ]
                }
              >
                Country
                <Text
                  style={
                    [
                      {
                        "fontFamily": "Inter_500Medium",
                      },
                      {
                        "fontSize": 16,
                      },
                      {
                        "color": "#fff",
                      },
                      {
                        "color": "#FFC93F",
                        "marginLeft": 4,
                      },
                    ]
                  }
                >
                   *
                </Text>
              </Text>
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessible={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  [
                    {
                      "alignItems": "center",
                      "borderColor": "#fff",
                      "borderRadius": 12,
                      "borderWidth": 1,
                      "color": "#fff",
                      "flexDirection": "row",
                      "fontSize": 16,
                      "justifyContent": "space-between",
                      "paddingBottom": 14,
                      "paddingLeft": 18,
                      "paddingRight": 18,
                      "paddingTop": 12,
                    },
                    false,
                  ]
                }
              >
                <Text
                  style={
                    [
                      {
                        "flex": 1,
                        "fontSize": 16,
                        "writingDirection": "ltr",
                      },
                      {
                        "color": "#6F6F6F",
                      },
                      {},
                    ]
                  }
                >
                  United States
                </Text>
                <RNSVGSvgView
                  align="xMidYMid"
                  bbHeight="8"
                  bbWidth="9"
                  fill="none"
                  focusable={false}
                  height={8}
                  meetOrSlice={0}
                  minX={0}
                  minY={0}
                  style={
                    [
                      {
                        "backgroundColor": "transparent",
                        "borderWidth": 0,
                      },
                      {
                        "flex": 0,
                        "height": 8,
                        "width": 9,
                      },
                    ]
                  }
                  vbHeight={8}
                  vbWidth={9}
                  width={9}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <RNSVGGroup
                    fill={null}
                    propList={
                      [
                        "fill",
                      ]
                    }
                  >
                    <RNSVGPath
                      d="M8.25181 0.943938L0.905813 0.943937C0.831439 0.94417 0.758536 0.964683 0.694952 1.00327C0.631368 1.04185 0.579511 1.09705 0.544963 1.16291C0.510416 1.22877 0.494484 1.30281 0.498885 1.37706C0.503285 1.4513 0.527852 1.52294 0.569938 1.58426L4.24294 6.88971C4.39516 7.10968 4.76165 7.10968 4.91428 6.88971L8.58728 1.58426C8.62979 1.52307 8.65473 1.4514 8.65937 1.37703C8.66401 1.30266 8.64818 1.22844 8.6136 1.16244C8.57902 1.09643 8.52701 1.04117 8.46323 1.00265C8.39944 0.964126 8.32632 0.94382 8.25181 0.943938Z"
                      fill={
                        {
                          "payload": 4286019447,
                          "type": 0,
                        }
                      }
                      propList={
                        [
                          "fill",
                        ]
                      }
                    />
                  </RNSVGGroup>
                </RNSVGSvgView>
              </View>
            </ViewManagerAdapter_ExpoLinearGradient>
          </View>
        </View>
        <View
          style={
            [
              {
                "marginBottom": 24,
                "width": "100%",
              },
              false,
            ]
          }
        >
          <View
            onLayout={[Function]}
            style={
              {
                "justifyContent": "center",
              }
            }
          >
            <ViewManagerAdapter_ExpoLinearGradient
              colors={
                [
                  4278979596,
                  4279637526,
                ]
              }
              style={
                {
                  "borderRadius": 12,
                  "position": "relative",
                  "zIndex": 20,
                }
              }
            >
              <Text
                style={
                  [
                    {
                      "fontFamily": "Inter_500Medium",
                    },
                    {
                      "fontSize": 16,
                    },
                    {
                      "color": "#fff",
                    },
                    {
                      "backgroundColor": "#161616",
                      "borderRadius": 10,
                      "color": "#fff",
                      "fontSize": 12,
                      "fontWeight": "normal",
                      "marginLeft": 16,
                      "paddingBottom": 1.6,
                      "paddingLeft": 7.2,
                      "paddingRight": 7.2,
                      "paddingTop": 1.6,
                      "position": "absolute",
                      "top": -12,
                      "zIndex": 20,
                    },
                  ]
                }
              >
                State
              </Text>
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessible={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  [
                    {
                      "alignItems": "center",
                      "borderColor": "#fff",
                      "borderRadius": 12,
                      "borderWidth": 1,
                      "color": "#fff",
                      "flexDirection": "row",
                      "fontSize": 16,
                      "justifyContent": "space-between",
                      "paddingBottom": 14,
                      "paddingLeft": 18,
                      "paddingRight": 18,
                      "paddingTop": 12,
                    },
                    false,
                  ]
                }
              >
                <Text
                  style={
                    [
                      {
                        "flex": 1,
                        "fontSize": 16,
                        "writingDirection": "ltr",
                      },
                      {
                        "color": "#6F6F6F",
                      },
                      {},
                    ]
                  }
                >
                  Colorado
                </Text>
                <RNSVGSvgView
                  align="xMidYMid"
                  bbHeight="8"
                  bbWidth="9"
                  fill="none"
                  focusable={false}
                  height={8}
                  meetOrSlice={0}
                  minX={0}
                  minY={0}
                  style={
                    [
                      {
                        "backgroundColor": "transparent",
                        "borderWidth": 0,
                      },
                      {
                        "flex": 0,
                        "height": 8,
                        "width": 9,
                      },
                    ]
                  }
                  vbHeight={8}
                  vbWidth={9}
                  width={9}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <RNSVGGroup
                    fill={null}
                    propList={
                      [
                        "fill",
                      ]
                    }
                  >
                    <RNSVGPath
                      d="M8.25181 0.943938L0.905813 0.943937C0.831439 0.94417 0.758536 0.964683 0.694952 1.00327C0.631368 1.04185 0.579511 1.09705 0.544963 1.16291C0.510416 1.22877 0.494484 1.30281 0.498885 1.37706C0.503285 1.4513 0.527852 1.52294 0.569938 1.58426L4.24294 6.88971C4.39516 7.10968 4.76165 7.10968 4.91428 6.88971L8.58728 1.58426C8.62979 1.52307 8.65473 1.4514 8.65937 1.37703C8.66401 1.30266 8.64818 1.22844 8.6136 1.16244C8.57902 1.09643 8.52701 1.04117 8.46323 1.00265C8.39944 0.964126 8.32632 0.94382 8.25181 0.943938Z"
                      fill={
                        {
                          "payload": 4286019447,
                          "type": 0,
                        }
                      }
                      propList={
                        [
                          "fill",
                        ]
                      }
                    />
                  </RNSVGGroup>
                </RNSVGSvgView>
              </View>
            </ViewManagerAdapter_ExpoLinearGradient>
          </View>
        </View>
        <View
          style={
            [
              {
                "marginBottom": 24,
                "width": "100%",
              },
              false,
            ]
          }
        >
          <View
            onLayout={[Function]}
            style={
              {
                "justifyContent": "center",
              }
            }
          >
            <ViewManagerAdapter_ExpoLinearGradient
              colors={
                [
                  4278979596,
                  4279637526,
                ]
              }
              style={
                {
                  "borderRadius": 12,
                  "position": "relative",
                  "zIndex": 20,
                }
              }
            >
              <Text
                style={
                  [
                    {
                      "fontFamily": "Inter_500Medium",
                    },
                    {
                      "fontSize": 16,
                    },
                    {
                      "color": "#fff",
                    },
                    {
                      "backgroundColor": "#161616",
                      "borderRadius": 10,
                      "color": "#fff",
                      "fontSize": 12,
                      "fontWeight": "normal",
                      "marginLeft": 16,
                      "paddingBottom": 1.6,
                      "paddingLeft": 7.2,
                      "paddingRight": 7.2,
                      "paddingTop": 1.6,
                      "position": "absolute",
                      "top": -12,
                      "zIndex": 20,
                    },
                  ]
                }
              >
                City
              </Text>
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessible={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  [
                    {
                      "alignItems": "center",
                      "borderColor": "#fff",
                      "borderRadius": 12,
                      "borderWidth": 1,
                      "color": "#fff",
                      "flexDirection": "row",
                      "fontSize": 16,
                      "justifyContent": "space-between",
                      "paddingBottom": 14,
                      "paddingLeft": 18,
                      "paddingRight": 18,
                      "paddingTop": 12,
                    },
                    false,
                  ]
                }
              >
                <Text
                  style={
                    [
                      {
                        "flex": 1,
                        "fontSize": 16,
                        "writingDirection": "ltr",
                      },
                      {
                        "color": "#6F6F6F",
                      },
                      {},
                    ]
                  }
                >
                  Denver
                </Text>
                <RNSVGSvgView
                  align="xMidYMid"
                  bbHeight="8"
                  bbWidth="9"
                  fill="none"
                  focusable={false}
                  height={8}
                  meetOrSlice={0}
                  minX={0}
                  minY={0}
                  style={
                    [
                      {
                        "backgroundColor": "transparent",
                        "borderWidth": 0,
                      },
                      {
                        "flex": 0,
                        "height": 8,
                        "width": 9,
                      },
                    ]
                  }
                  vbHeight={8}
                  vbWidth={9}
                  width={9}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <RNSVGGroup
                    fill={null}
                    propList={
                      [
                        "fill",
                      ]
                    }
                  >
                    <RNSVGPath
                      d="M8.25181 0.943938L0.905813 0.943937C0.831439 0.94417 0.758536 0.964683 0.694952 1.00327C0.631368 1.04185 0.579511 1.09705 0.544963 1.16291C0.510416 1.22877 0.494484 1.30281 0.498885 1.37706C0.503285 1.4513 0.527852 1.52294 0.569938 1.58426L4.24294 6.88971C4.39516 7.10968 4.76165 7.10968 4.91428 6.88971L8.58728 1.58426C8.62979 1.52307 8.65473 1.4514 8.65937 1.37703C8.66401 1.30266 8.64818 1.22844 8.6136 1.16244C8.57902 1.09643 8.52701 1.04117 8.46323 1.00265C8.39944 0.964126 8.32632 0.94382 8.25181 0.943938Z"
                      fill={
                        {
                          "payload": 4286019447,
                          "type": 0,
                        }
                      }
                      propList={
                        [
                          "fill",
                        ]
                      }
                    />
                  </RNSVGGroup>
                </RNSVGSvgView>
              </View>
            </ViewManagerAdapter_ExpoLinearGradient>
          </View>
        </View>
        <View
          style={
            [
              {
                "marginBottom": 16,
              },
              {
                "width": "100%",
              },
            ]
          }
        >
          <ViewManagerAdapter_ExpoLinearGradient
            colors={
              [
                4293899034,
                4294953798,
              ]
            }
            style={
              [
                {
                  "backgroundColor": "rgba(56, 52, 42, 0.38)",
                },
                {
                  "borderRadius": 10,
                },
              ]
            }
          >
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                [
                  {
                    "alignItems": "center",
                    "borderWidth": 2,
                    "color": "#000",
                    "display": "flex",
                    "flexDirection": "row",
                    "justifyContent": "center",
                    "paddingBottom": 15,
                    "paddingLeft": 24,
                    "paddingRight": 24,
                    "paddingTop": 13,
                  },
                  {
                    "borderColor": "#ffd978",
                  },
                  false,
                  false,
                  false,
                  {
                    "borderRadius": 10,
                  },
                  undefined,
                ]
              }
            >
              <Text
                style={
                  [
                    {
                      "fontFamily": "Inter_500Medium",
                    },
                    {
                      "fontSize": 16,
                    },
                    {
                      "color": "#fff",
                    },
                    [
                      {
                        "color": "#000",
                      },
                      false,
                      {
                        "fontSize": 14,
                        "lineHeight": 20,
                      },
                    ],
                  ]
                }
              >
                Next: Set Profile Images
              </Text>
            </View>
          </ViewManagerAdapter_ExpoLinearGradient>
        </View>
      </View>
    </View>
  </RNCSafeAreaView>
</View>
`;
